incubator/hydra/values.yaml

```diff
@@ -0,0 +1,103 @@
+image: "oryd/hydra"
+imageTag: "v0.8.6"
+imagePullPolicy: "Always"
```

Comment:
@unguiculus Hydra is currently pre-release (pre-v1, though generally stable) and pushes out frequent security or other patches. Thus, the intention here was to allow image tags to stabilize (i.e. a v1 tag that is v1.0-v1.X) while still pulling the Docker images, but it's not strictly required.

In the end, there's a reason this is configurable 😛.